### ISSUE TITLE
SVG tag is stripped out

### ISSUE THREAD

**Issue by EternallLight** [MurhafSousli/ngx-sharebuttons#153]
**I'm submitting a ...**  (check one with "x")
```
[x ] bug report => search for a similar issue before submitting
[ ] feature request
[ ] question
```

**Minimal usage of the plugin**

Hello,

I've updated to ngx-sharebuttons 2.2.1 (on Angular 4) from ng2-sharebuttons 1.0.6 (Angular 2), and my custom buttons stopped working. Inspection shows that for some reason, the <svg> tags is not rendered in <share-button> inner template. 

I have code like this:
```
// in ngOnInit() {
const fbTemplate = '<svg class="i-facebook-square"><use xlink:href="/assets/images/icons.svg#i-facebook-square"></use></svg>';
this.fbButton = new ShareButton(ShareProvider.FACEBOOK, fbTemplate, 'facebook');
```

If I insert such an SVG inside any template around the buttons, it renders well, so it does not seem to be a particular project's issue.

Please, check.

**Comment by EternallLight** [comment]
As I can see from source code, you used to use renderer to set innerHTML

```
/** Set button template */
        this.renderer.setElementProperty(this.btn.nativeElement, 'innerHTML', this.button.template);
```

But after update it is made with template syntax:
```
<div class="sb-template" [innerHtml]="button.template"></div>
```

This issue is described here: https://stackoverflow.com/questions/40727183/angular-2-svg-not-rendering

If you decide to fix this, you may want to add a property like "disableHtmlFilter: boolean".

**Comment by MurhafSousli** [comment]
Hi @EternallLight, This will fixed in `v4.0.0`, stay tuned!
